charts/utils/Chart.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v2
+name: saritasa-utils
+
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.0.1
+
+maintainers:
+  - url: https://www.saritasa.com/
+    name: Saritasa
+    email: nospam@saritasa.com
+
+description: |
+  A Helm chart for utils resources.
+
+  Implements:
+  - port-forward deployments for external resources
+  - utils deployments (db clients/network/etc) with extended network capabilities
+  - RBAC
+
+  ## `example usage with argocd`
+
+  Install the chart:
+
+  ```
+  helm repo add saritasa https://saritasa-nest.github.io/saritasa-devops-helm-charts/
+  ```
+
+  Make the argocd manifest
+
+  ```yaml
+  apiVersion: argoproj.io/v1alpha1
+  kind: Application
+  metadata:
+    name: utils
+    namespace: argo-cd
+    finalizers:
+    - resources-finalizer.argocd.argoproj.io
+    annotations:
+      argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+      argocd.argoproj.io/sync-wave: "1000"
+  spec:
+    destination:
+      server: https://kubernetes.default.svc
+      namespace: utils
+    project: default
+    source:
+      chart: saritasa-utils
+      helm:
+        values: |
+          client: saritasa
+          environment: rocks
+          rbac:
+            developers: true
+          deployments:
+            entities:
+              - name: mssql
+                image: mcr.microsoft.com/mssql/server:2019-latest
```

Comment:
Where is the list with images for all db engines?

Reply:
there is no restrictions for image, you can use any, even network tools

Reply:
we still should have a config valid for default services - like psql, mysql, mssql, mongo, redis that we use. Also keep in mind that some images can't be used as they don't have shell and just provide the entrypoint as the executable like `mysql` - these images won't be usable for our scenario.